decidim-bulletin-board-client-ruby/Gemfile

```diff
@@ -1,7 +1,9 @@
-source "https://rubygems.org"
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
 
 # Specify your gem's dependencies in decidim-bulletin_board.gemspec
 gemspec
 
-gem "rake", "~> 12.0"
-gem "rspec", "~> 3.0"
+gem 'rake', '~> 12.0'
+gem 'rspec', '~> 3.0'
```

Comment:
for the web app we are using [the Decidim rubocop rules](https://github.com/decidim/decidim-bulletin-board/blob/master/.rubocop.yml#L1), maybe we could use them here too. What do you think?